hs_core/views/__init__.py

```diff
@@ -458,10 +458,10 @@ class FilterForm(forms.Form):
 
 @processor_for('my-resources')
 def my_resources(request, page):
-    # import sys
-    # sys.path.append("/home/docker/pycharm-debug")
-    # import pydevd
-    # pydevd.settrace('172.17.42.1', port=21000, suspend=False)
+    import sys
```

Comment:
this "import sys" is used by pycharm-debug and is an artifact for keeping debug traces in, hence is not needed. Just revert back to where it was in the latest develop, in fact, there is no change needed in view/__init__.py from this branch.